charts/clickhouse/templates/clickhouse-instance/clickhouse-instance.yaml

```
@@ -5,6 +5,9 @@ metadata:
   namespace: {{ include "clickhouse.namespace" . }}
   labels:
     {{- include "clickhouse.labels" . | nindent 4 }}
+    {{- if .Values.labels }}
```

yamllint

[warning] wrong indentation: expected 2 but found 4

(indentation)